### DISCUSSION TITLE
iCloud Cal to Google Cal Sync?

### DISCUSSION THREAD

**Discussion by SP3C5** [phw198/OutlookGoogleCalendarSync#2001] [Ideas]
I absolutely love using the Outlook to Google Calendar Sync app, and it's been working great for syncing my iCloud calendar to Google. Right now, I’m using Outlook mostly as a "buffer" to connect iCloud and Google, syncing my iCloud calendar to Outlook via the iCloud for Windows app.

I was wondering if there’s any interest in, or possibility of, a new feature or even a standalone app that could directly sync between iCloud and Google, skipping Outlook altogether? It would be amazing to streamline this process a bit further.

Thanks so much for the awesome app!

**Comment by phw198** [comment]
Hi @SP3C5 Did you post on reddit about this the other day? Unfortunately, what little time I do have is going towards removing the need for the Outlook client, as the New Outlook doesn't support applications using COM interoperation. However, I'm not sure it's possible to set up an iCloud account in Outlook on the Web? If not, this means that using Outlook as a "proxy" for iCloud calendars will die along with the classic Outlook.

What I have been considering is the ability to sync with an ICS URL (see #1893) - a lot of corporate Outlook accounts are probably going to block OAuth from OGCS, but oddly, it still seems possible to publish an ICS feed. If you can use an iCloud ICS, then this would give you one-way sync at least.

Being quite honest though, native iCloud sync is probably not going to happen - just not enough hours in the day!
